src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1637,6 +1647,7 @@ void goto_instrument_parse_optionst::help()
     HELP_REMOVE_CONST_FUNCTION_POINTERS
     " --add-library                add models of C library functions\n"
     " --model-argc-argv <n>        model up to <n> command line arguments\n"
+    " --remove-function-body <f>   remove the implementation of function <f>\n"
```

Comment:
Could it also be clearer that here you can provide multiple `f`s to remove multiple functions.